controllers/users.go

```diff
@@ -0,0 +1,102 @@
+package controllers
+
+import (
+	"costmanagement/models"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+)
+
+type CreateUserInput struct {
+	Name     string `json:"name" binding:"required"`
+	Email    string `json:"email" binding:"required"`
+	Phone    string `json:"phone" binding:"required"`
+	Password string `json:"password" binding:"required"`
+}
+
+type UpdateUserInput struct {
+	Name     string `json:"name" `
+	Email    string `json:"email" `
+	Phone    string `json:"phone" `
+	Password string `json:"password" `
+}
+
```

Comment:
move these struct inside models folder, create a subfolder `dtos` inside models and move there. Make controllers fresh

Reply:
change done